x-pack/plugins/infra/server/routes/snapshot/lib/transform_metrics_ui_response.test.ts

```diff
@@ -5,28 +5,55 @@
  * 2.0.
  */
 
+import {
+  InfraTimerangeInput,
+  MetricsAPIRequest,
+  SnapshotRequest,
+} from '../../../../common/http_api';
+import moment from 'moment';
 import { transformMetricsApiResponseToSnapshotResponse } from './transform_metrics_ui_response';
 
 jest.mock('./apply_metadata_to_last_path', () => ({
   applyMetadataToLastPath: (series: any) => [{ label: series.id }],
 }));
 
-const now = 1630597319235;
+const now = moment('2020-01-01T00:00:00Z').add(5, 'minute').valueOf();
+
+const timerange: InfraTimerangeInput = {
+  from: moment('2020-01-01T00:00:00Z').valueOf(),
+  to: now,
+  interval: '1m',
+};
+
+const snapshotApiRequest: SnapshotRequest = {
+  metrics: [{ type: 'cpu' }],
+  includeTimeseries: false,
+  timerange,
+  groupBy: [],
+  nodeType: 'host',
+  sourceId: '',
+};
+
+const metricsApiRequest: MetricsAPIRequest = {
+  metrics: [
+    {
+      id: 'cpu',
+      aggregations: { cpu: { avg: { field: 'system.cpu.user.pct' } } },
```

Comment:
I saw that we have `user` and `system` in the [model](https://github.com/elastic/kibana/blob/main/x-pack/plugins/infra/common/inventory_models/host/metrics/snapshot/cpu.ts) - when do we set system and when user? Do we need a separate case for `system` or it should be the same behavior? 🤔

Reply:
I'm not sure if I understood the question, but we use `system.cpu.user.pct`, `system.cpu.system.pct` and `system.cpu.cores` to calculate the CPU metric, used a standard or metric.

But both are used individually as custom metric in the infra inventory flyout: https://github.com/elastic/kibana/blob/main/x-pack/plugins/infra/public/pages/metrics/inventory_view/components/node_details/tabs/metrics/metrics.tsx#L112

Reply:
Ah Ok, thanks for the explanation!